examples/suit_update/README.md

```diff
@@ -140,37 +173,51 @@ If the Border Router is already set up when opening the terminal you should get
 
     suit_coap: started.
 
-The global address would be: "2001:db8::7b7e:3255:1313:8d96". The address you see
-will be different according to your device and the chosen prefix. In this case
-the RIOT node will be reachable via its global address.
+Here the global IPv6 is `2001:db8::7b7e:3255:1313:8d96`.
+**The address will be different according to your device and the chosen prefix**.
+In this case the RIOT node can be reached from the host using its global address:
+
+    $ ping6 2001:db8::7b7e:3255:1313:8d96
+
+### Setup aiocoap-fileserver
+[setup-aiocoap-fileserver]: #Setup-aiocoap-fileserver
+
+`aiocoap-fileserver` is used for hosting firmwares available for updates. Device
+retrieve the new firmware using the CoAP protocol.
 
-## Setup fw-server
+Clone the aiocoap repository:
 
-In this examples we are using aiocoap-fileserver. to start aiocoap-fileserver:
+    $ git clone https://github.com/chrysn/aiocoap.git ..
 
-    $ mkdir ${RIOTBASE}/coaproot
-    $ <PATH>/aiocoap-fileserver ${RIOTBASE}/coaproot
+Start `aiocoap-fileserver`:
 
-If aiocoap was cloned and built from source aiocoap-fileserver will be located
-at <AIOCOAP_BASE_DIR>/aiocoap/contrib.
+    $ mkdir -p coaproot
+    $ ../aiocoap/contrib/aiocoap-fileserver coaproot
 
-# Update IoT device
+Keep the server running in the terminal.
 
-## Build and publish firmware update
+## Perform an update
+[update]: #Perform-an-update
+
+### Build and publish the firmware update
+[update-build-publish]: #Build-and-publish-the-firmware-update
 
 Currently, the build system assumes that it can publish files by simply copying
-them to a configurable folder. For this example, aiocoap-fileserver will then
-serve the files via CoAP.
+them to a configurable folder.
+
+For this example, aiocoap-fileserver serves the files via CoAP.
+
+- To publish an update for a node in wired mode (behind ethos):
 
-Manifests and image files will be copied to $(SUIT_COAP_FSROOT)/$(SUIT_COAP_BASEPATH).
+    $ BOARD=samr21-xpro SUIT_COAP_SERVER=[2001:db8::1] make -C examples/suit_update suit/publish
```

Comment:
Because of the bullet point this doesn't render as a code command, maybe enclose with ``. Same for commands below